WinPort/src/Backend/TTY/TTYOutput.cpp

```diff
@@ -188,6 +189,7 @@ TTYOutput::~TTYOutput()
 		}
 		Format(ESC "[0m" ESC "[?1049l" ESC "[?47l" ESC "8" ESC "[?2004l" "\r\n");
 		Format(ESC "[?9001l"); // win32-input-mode off
+		Format(ESC "[=0;1u"); // kovidgoyal's kitty mode off
```

Comment:
Likely this will leave '0;1u' after far2l closing in terminal that doesnt support Kitty

Reply:
fixed